qiita_pet/handlers/user_handlers.py

```diff
@@ -185,3 +186,10 @@ def post(self):
 
         self.render("user_messages.html",
                     messages=self.current_user.messages())
+
+
+class UserJobs(BaseHandler):
+    @authenticated
+    def get(self):
+        response = user_jobs_get_req(self.current_user)
```

Comment:
this executes as a transaction and has the potential to block as it is performing IO. Should this method be a tornado `coroutine` instead?